satpy/tests/reader_tests/test_ahi_hrit.py

```diff
@@ -182,9 +183,9 @@ def test_get_area_def(self):
                                 segno=case['segno'], numseg=case['numseg'])
             reader = self._get_reader(mda=mda,
                                       filename_info={'area': case['area']})
-
-            self.assertTupleEqual(reader._get_area_def().area_extent,
-                                  case['extent'])
+            area = reader._get_area_def()
```

Comment:
Just for sanity and future clarity can this use `reader.get_area_def(<some id>)` instead?

Reply:
Yes, changed it.